src/components/AdminPage/AdminNavigation/AdminMenuPages.js

```diff
@@ -0,0 +1,41 @@
+import { NavLink } from "react-router-dom";
+import { Image, Button } from "react-bootstrap";
+import ProfileIcon from "../../../images/myProfile-dark.png";
+import MenuIcon from "../../../images/menuAdmin-dark.png";
+import Members from "../../../images/members-dark.png";
+
+const Pages = props => {
+	return (
+		<div
+			className=' mb-3 p-2 active-page fs-6 bg-light '
+			style={{ fontFamily: "Roboto", maxHeight: "60px" }}>
+			<NavLink
+				to={props.source}
+				className='text-decoration-none border-3 d-flex w-100'
+				activeclassname='active'>
+				<Button className='text-decoration-none bg-transparent shadow-none border-none fw-bold'>
+					<Image src={props.icon} className='active-page d-inline'></Image>
+					<div className='text-secondary d-inline mx-4'>{props.name}</div>
+				</Button>
+			</NavLink>
+		</div>
+	);
+};
```

Comment:
please create separate file for Pages Component and use { source, name, icon, key } instead of just props in line 7